### PR TITLE
Fixes #6231: Document that when we reduce the run interval of nodes, \"no data\" may be wrongly reported

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/6231

### CODE REVIEW

File: 4_advanced_usage/10_node_management.txt [hunk 1]
```diff
@@ -25,7 +25,13 @@ a full reinstallation of the Rudder Server.
...
 
+==== Change the agent run schedule
 
+By default, the agent runs on all nodes every 5 minutes. You can modify this value in
+*Administration* -> *Settings* -> *Agent Run Schedule*, as well as the "splay time"
+across nodes (a random delay that alters scheduled run time, intended to spread
+load across nodes).
 
-
+When reducing notably the run interval length, reporting can be in 'Missing' state
```
Comment (fanf) [inline 4_advanced_usage/10_node_management.txt]: Shouldn't it be "no report" in place of missing ?

**Comment by amousset** [comment]
PR updated

File: 4_advanced_usage/10_node_management.txt (hunk 1 above)
Comment (amousset) [inline 4_advanced_usage/10_node_management.txt]: Fixed, thanks.

**Comment by amousset** [comment]
ping @fanf

File: 4_advanced_usage/10_node_management.txt [hunk 2]
```diff
@@ -25,7 +25,13 @@ a full reinstallation of the Rudder Server.
...
+==== Change the agent run schedule
 
+By default, the agent runs on all nodes every 5 minutes. You can modify this value in
+*Administration* -> *Settings* -> *Agent Run Schedule*, as well as the "splay time"
+across nodes (a random delay that alters scheduled run time, intended to spread
+load across nodes).
 
-
+When reducing notably the run interval length, reporting can be in 'No report' state
+until the next run of the agent, which can take up to the previous (longer) interval.
```
Comment (jooooooon) [inline 4_advanced_usage/10_node_management.txt]: I like the text, but could we have it in a special "warning" section to highlight it?

**Comment by amousset** [comment]
PR updated

**Pull request merged**